### PR TITLE
[Solutions] Commerce chatbot

### PR DESCRIPTION
**Pull request by iainfoulds**

### CODE REVIEW

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/sample-solutions/event-driven-apps/commerce-chatbot.md](https://github.com/iainfoulds/architecture-center/blob/commerce-chatbot/docs/sample-solutions/event-driven-apps/commerce-chatbot.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/sample-solutions/event-driven-apps/commerce-chatbot?branch=pr-en-us-584) |
[docs/sample-solutions/event-driven-apps/media/commerce-chatbot/architecture-commerce-chatbot.png](https://github.com/iainfoulds/architecture-center/blob/commerce-chatbot/docs/sample-solutions/event-driven-apps/media/commerce-chatbot/architecture-commerce-chatbot.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/sample-solutions/event-driven-apps/commerce-chatbot?branch=pr-en-us-584) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C15%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201806151557245294-584%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=bK1jlbTn7FeChJgYObAwmt6hLPC8f5i82caaWsdlcoo%3D&st=2018-06-15T15%3A53%3A29Z&se=2018-07-16T15%3A58%3A29Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bennage** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/sample-solutions/event-driven-apps/index.md](https://github.com/iainfoulds/architecture-center/blob/commerce-chatbot/docs/sample-solutions/event-driven-apps/index.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/azure/architecture/sample-solutions/event-driven-apps/index?branch=pr-en-us-584) | [Details](#user-content-docs%2Fsample-solutions%2Fevent-driven-apps%2Findex.md)
[docs/sample-solutions/event-driven-apps/bot-service.svg](https://github.com/iainfoulds/architecture-center/blob/commerce-chatbot/docs/sample-solutions/event-driven-apps/bot-service.svg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/sample-solutions/event-driven-apps/bot-service.svg?branch=pr-en-us-584) |
[docs/sample-solutions/event-driven-apps/commerce-chatbot.md](https://github.com/iainfoulds/architecture-center/blob/commerce-chatbot/docs/sample-solutions/event-driven-apps/commerce-chatbot.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/sample-solutions/event-driven-apps/commerce-chatbot?branch=pr-en-us-584) |
[docs/sample-solutions/event-driven-apps/media/commerce-chatbot/architecture-commerce-chatbot.png](https://github.com/iainfoulds/architecture-center/blob/commerce-chatbot/docs/sample-solutions/event-driven-apps/media/commerce-chatbot/architecture-commerce-chatbot.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/sample-solutions/event-driven-apps/commerce-chatbot?branch=pr-en-us-584) |
[docs/sample-solutions/toc.yml](https://github.com/iainfoulds/architecture-center/blob/commerce-chatbot/docs/sample-solutions/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/sample-solutions/event-driven-apps/index?branch=pr-en-us-584) |

<a name="docs%2Fsample-solutions%2Fevent-driven-apps%2Findex.md"></a>
### [docs/sample-solutions/event-driven-apps/index.md](https://github.com/iainfoulds/architecture-center/blob/commerce-chatbot/docs/sample-solutions/event-driven-apps/index.md)
 - **Line 0**: **[Warning]** Invalid file link:(~/sample-solutions/event-driven-apps/media/bot-service.svg).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C15%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201806151628145984-584%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=tEGFPeRtEMgRTJqPRiZAVReveEP%2BtTzr29vzEGe6mYA%3D&st=2018-06-15T16%3A24%3A22Z&se=2018-07-16T16%3A29%3A22Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/sample-solutions/event-driven-apps/commerce-chatbot.md](https://github.com/iainfoulds/architecture-center/blob/commerce-chatbot/docs/sample-solutions/event-driven-apps/commerce-chatbot.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/sample-solutions/event-driven-apps/commerce-chatbot?branch=pr-en-us-584) |
[docs/sample-solutions/event-driven-apps/index.md](https://github.com/iainfoulds/architecture-center/blob/commerce-chatbot/docs/sample-solutions/event-driven-apps/index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/sample-solutions/event-driven-apps/index?branch=pr-en-us-584) |
[docs/sample-solutions/event-driven-apps/media/bot-service.svg](https://github.com/iainfoulds/architecture-center/blob/commerce-chatbot/docs/sample-solutions/event-driven-apps/media/bot-service.svg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/sample-solutions/event-driven-apps/index?branch=pr-en-us-584) |
[docs/sample-solutions/event-driven-apps/media/commerce-chatbot/architecture-commerce-chatbot.png](https://github.com/iainfoulds/architecture-center/blob/commerce-chatbot/docs/sample-solutions/event-driven-apps/media/commerce-chatbot/architecture-commerce-chatbot.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/sample-solutions/event-driven-apps/commerce-chatbot?branch=pr-en-us-584) |
[docs/sample-solutions/toc.yml](https://github.com/iainfoulds/architecture-center/blob/commerce-chatbot/docs/sample-solutions/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/sample-solutions/event-driven-apps/index?branch=pr-en-us-584) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C15%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201806151632212747-584%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ejU%2BnbStQkZiG3IILnXES98qDsVXw5D5KnMGDcyQGds%3D&st=2018-06-15T16%3A28%3A31Z&se=2018-07-16T16%3A33%3A31Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/sample-solutions/event-driven-apps/commerce-chatbot.md](https://github.com/iainfoulds/architecture-center/blob/commerce-chatbot/docs/sample-solutions/event-driven-apps/commerce-chatbot.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/sample-solutions/event-driven-apps/commerce-chatbot?branch=pr-en-us-584) |
[docs/sample-solutions/event-driven-apps/index.md](https://github.com/iainfoulds/architecture-center/blob/commerce-chatbot/docs/sample-solutions/event-driven-apps/index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/sample-solutions/event-driven-apps/index?branch=pr-en-us-584) |
[docs/sample-solutions/event-driven-apps/media/bot-service.svg](https://github.com/iainfoulds/architecture-center/blob/commerce-chatbot/docs/sample-solutions/event-driven-apps/media/bot-service.svg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/sample-solutions/event-driven-apps/index?branch=pr-en-us-584) |
[docs/sample-solutions/event-driven-apps/media/commerce-chatbot/architecture-commerce-chatbot.png](https://github.com/iainfoulds/architecture-center/blob/commerce-chatbot/docs/sample-solutions/event-driven-apps/media/commerce-chatbot/architecture-commerce-chatbot.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/sample-solutions/event-driven-apps/commerce-chatbot?branch=pr-en-us-584) |
[docs/sample-solutions/event-driven-apps/templates/commerce-chatbot.json](https://github.com/iainfoulds/architecture-center/blob/commerce-chatbot/docs/sample-solutions/event-driven-apps/templates/commerce-chatbot.json) | :white_check_mark:Succeeded | |
[docs/sample-solutions/toc.yml](https://github.com/iainfoulds/architecture-center/blob/commerce-chatbot/docs/sample-solutions/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/sample-solutions/event-driven-apps/index?branch=pr-en-us-584) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C15%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201806152004523202-584%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Q0bK6sb4Ve%2FBRwA5ZJ%2BTSND%2FT5Rd8AlLo%2FDv7ldW9W0%3D&st=2018-06-15T20%3A00%3A59Z&se=2018-07-16T20%3A05%3A59Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/sample-solutions/event-driven-apps/commerce-chatbot.md](https://github.com/iainfoulds/architecture-center/blob/commerce-chatbot/docs/sample-solutions/event-driven-apps/commerce-chatbot.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/sample-solutions/event-driven-apps/commerce-chatbot?branch=pr-en-us-584) |
[docs/sample-solutions/event-driven-apps/index.md](https://github.com/iainfoulds/architecture-center/blob/commerce-chatbot/docs/sample-solutions/event-driven-apps/index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/sample-solutions/event-driven-apps/index?branch=pr-en-us-584) |
[docs/sample-solutions/event-driven-apps/media/bot-service.svg](https://github.com/iainfoulds/architecture-center/blob/commerce-chatbot/docs/sample-solutions/event-driven-apps/media/bot-service.svg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/sample-solutions/event-driven-apps/index?branch=pr-en-us-584) |
[docs/sample-solutions/event-driven-apps/media/commerce-chatbot/architecture-commerce-chatbot.png](https://github.com/iainfoulds/architecture-center/blob/commerce-chatbot/docs/sample-solutions/event-driven-apps/media/commerce-chatbot/architecture-commerce-chatbot.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/sample-solutions/event-driven-apps/commerce-chatbot?branch=pr-en-us-584) |
[docs/sample-solutions/event-driven-apps/templates/commerce-chatbot.json](https://github.com/iainfoulds/architecture-center/blob/commerce-chatbot/docs/sample-solutions/event-driven-apps/templates/commerce-chatbot.json) | :white_check_mark:Succeeded | |
[docs/sample-solutions/toc.yml](https://github.com/iainfoulds/architecture-center/blob/commerce-chatbot/docs/sample-solutions/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/sample-solutions/event-driven-apps/index?branch=pr-en-us-584) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C15%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201806152039076697-584%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=DRV4H%2FJXVecGeM9jocOztG0AzbavyqQNCj9KneLQuf8%3D&st=2018-06-15T20%3A35%3A17Z&se=2018-07-16T20%3A40%3A17Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/sample-solutions/event-driven-apps/commerce-chatbot.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+* View restaurant take-out menu and order food
+* Check hotel availability and reserve a room
+* Search available photos and order prints
+
+## Architecture diagram
+
+The solution diagram below is an example of this solution:
+
+![Architecture overview of the Azure components involved in a conversational chatbot][architecture]
```
Comment (MikeWasson) [inline docs/sample-solutions/event-driven-apps/commerce-chatbot.md]: This diagram doesn't use the standard Azure colors, should use the standard icon set.

Comment (iainfoulds) [reply docs/sample-solutions/event-driven-apps/commerce-chatbot.md]: All artwork for these solutions is due to go through an artwork vendor for consistency with icon set. @david-stanford has more info.

**Pull request merged**